Modules/_opcode.c

```diff
@@ -425,15 +418,15 @@ opcode_functions[] =  {
 };
 
 int
-_opcode_exec(PyObject *m) {
+_Py_opcode_exec(PyObject *m) {
```

Comment:
Why this name change? It looks to me it's only used in L429, right?

Reply:
smelly was complaining about this.

Reply:
Ah, it's probably because it's not declared as `static`. It should be a file local function.

Reply:
I'll try that.